src/components/application_manager/src/policies/policy_handler.cc

```diff
@@ -1592,6 +1592,11 @@ uint32_t PolicyHandler::TimeoutExchangeMSec() const {
 
 void PolicyHandler::OnExceededTimeout() {
   POLICY_LIB_CHECK_VOID();
+
+  std::for_each(listeners_.begin(),
+                listeners_.end(),
+                std::mem_fn(&PolicyHandlerObserver::OnPTUTimeoutExceeded));
```

Comment:
@mked-luxoft why do we need mem_fn ? why not just &PolicyHandlerObserver::OnPTUTimeoutExceeded ?